pyomo/dae/contset.py

```diff
@@ -247,3 +248,54 @@ def construct(self, values=None):
         self._fe = sorted(self)
         timer.report()
 
+    def find_nearest_index(self, target, tolerance=None):
+        """ Returns the index of the nearest point in the 
+        :py:class:`ContinuousSet <pyomo.dae.ContinuousSet>`.
+
+        If a tolerance is specified, the index will only be returned
+        if the distance between the target and the closest point is
+        less than or equal to that tolerance. If the tolerance is less
+        than half the minimum spacing between points in the set, the
+        "closest-point-within-tolerance" will be unique.
```

Comment:
This last sentence seems like it no longer applies: the function should always return a unique (and deterministic) value.